docker/mongodb-kubernetes-tests/tests/upgrades/appdb_tls_operator_upgrade_v1_32_to_mck.py

```diff
@@ -127,8 +128,11 @@ def test_downscale_latest_official_operator(namespace: str):
 
 
 @mark.e2e_appdb_tls_operator_upgrade_v1_32_to_mck
-def test_upgrade_operator(default_operator: Operator):
-    default_operator.assert_is_running()
+def test_upgrade_operator(namespace: str, operator_installation_config: dict[str, str]):
+    operator = get_default_operator(
+        namespace, operator_installation_config=operator_installation_config, apply_crds_first=True
+    )
+    operator.assert_is_running()
```

Comment:
In #63 this test [was failing](https://parsley.mongodb.com/test/mongodb_kubernetes_e2e_multi_cluster_om_appdb_e2e_operator_upgrade_appdb_tls_patch_5b9d9814fae5fdd48c46a062b014a118cc22f062_68122894d40e5000075b6850_25_04_30_13_41_41/4/fbf31cca5414a6c2dfb8f578f2b42350?bookmarks=0,94&shareLine=0).

Reply:
Oh yes, we need to ensure we have the multicluster operator name set.

Reply:
Should be fixed in https://github.com/mongodb/mongodb-kubernetes/pull/127/commits/628a130deeb99ce7a78cc96b6539bc42edbcc133 . Running a patch [here](https://spruce.mongodb.com/version/6825b8cf68c7860007d34482/).

Reply:
Should be fixed by correctly installing the multi-cluster operator on upgrade: https://github.com/mongodb/mongodb-kubernetes/pull/127/commits/af513fd57ab3c9f605d6076712d2dda311b2b4f6 . Patch here: https://spruce.mongodb.com/version/6826141bbdf55e00077df7a9